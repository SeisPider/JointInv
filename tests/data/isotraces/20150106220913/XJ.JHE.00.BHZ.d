SURF96 R U X -1 90.00000 3.77036 0.23035
SURF96 R U X -1 85.00000 3.74590 0.21474
SURF96 R U X -1 75.00000 3.77548 0.19248
SURF96 R U X -1 70.00000 3.74187 0.17647
SURF96 R U X -1 65.00000 3.68522 0.15894
SURF96 R U X -1 60.00000 3.65837 0.14458
SURF96 R U X -1 55.00000 3.78228 0.14166
SURF96 R U X -1 42.00000 3.38353 0.08657
SURF96 R U X -1 40.00000 3.43215 0.08484
SURF96 R U X -1 38.00000 3.38997 0.07863
SURF96 R U X -1 36.00000 3.32871 0.07182
SURF96 R U X -1 34.00000 3.27463 0.06564
SURF96 R U X -1 32.00000 3.19475 0.05880
SURF96 R U X -1 29.00000 3.09948 0.05016
SURF96 R U X -1 28.00000 3.13262 0.04947
SURF96 R U X -1 27.00000 3.17119 0.04889
SURF96 R U X -1 26.00000 3.18690 0.04754
