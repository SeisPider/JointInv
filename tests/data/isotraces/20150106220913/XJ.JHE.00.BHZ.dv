MFT96 R U T -1 91.022    3.7197    1.0000    5554.1     0.4047E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 89.043    3.7314    1.0000    5554.1     0.4506E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 87.149    3.7424    1.0000    5554.1     0.4949E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 85.333    3.7526    1.0000    5554.1     0.5370E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 83.592    3.7621    1.0000    5554.1     0.5764E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 81.920    3.7709    1.0000    5554.1     0.6127E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 80.314    3.7789    1.0000    5554.1     0.6458E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 78.769    3.7862    1.0000    5554.1     0.6757E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 77.283    3.7928    1.0000    5554.1     0.7027E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 75.852    3.7987    1.0000    5554.1     0.7274E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 74.473    3.8039    1.0000    5554.1     0.7504E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 73.143    3.8084    1.0000    5554.1     0.7729E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 71.860    3.8123    1.0000    5554.1     0.7959E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 70.621    3.8154    1.0000    5554.1     0.8206E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 69.424    3.8178    1.0000    5554.1     0.8481E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 68.267    3.8196    1.0000    5554.1     0.8791E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 67.148    3.8208    1.0000    5554.1     0.9137E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 66.065    3.8213    1.0000    5554.1     0.9517E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 65.016    3.8212    1.0000    5554.1     0.9919E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 64.000    3.8204    1.0000    5554.1     0.1033E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 63.015    3.8191    1.0000    5554.1     0.1072E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 62.061    3.8171    1.0000    5554.1     0.1108E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 61.134    3.8146    1.0000    5554.1     0.1137E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 60.235    3.8116    1.0000    5554.1     0.1158E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 59.362    3.8080    1.0000    5554.1     0.1167E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 58.514    3.8038    1.0000    5554.1     0.1164E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 57.690    3.7992    1.0000    5554.1     0.1147E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 56.889    3.7941    1.0000    5554.1     0.1115E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 56.110    3.7884    1.0000    5554.1     0.1068E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 55.351    3.7824    1.0000    5554.1     0.1008E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 54.613    3.7758    1.0000    5554.1     0.9368E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 53.895    3.7689    1.0000    5554.1     0.8563E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 53.195    3.7615    1.0000    5554.1     0.7699E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 52.513    3.7538    1.0000    5554.1     0.6816E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 51.848    3.7457    1.0000    5554.1     0.5955E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 51.200    3.7372    1.0000    5554.1     0.5155E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 50.568    3.7283    1.0000    5554.1     0.4444E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 49.951    3.7192    1.0000    5554.1     0.3832E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 49.349    3.7097    1.0000    5554.1     0.3296E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 48.762    3.7000    1.0000    5554.1     0.2785E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 48.188    3.6900    1.0000    5554.1     0.2228E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 47.628    3.6797    1.0000    5554.1     0.1554E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 47.080    3.6692    1.0000    5554.1     0.7083E-07   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 46.545    3.6584    1.0000    5554.1     0.4391E-07   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 46.022    3.6475    1.0000    5554.1     0.1748E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 45.511    3.6363    1.0000    5554.1     0.3349E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 45.011    3.6250    1.0000    5554.1     0.5218E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 44.522    3.6135    1.0000    5554.1     0.7339E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 44.043    3.6019    1.0000    5554.1     0.9692E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 43.574    3.5902    1.0000    5554.1     0.1225E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 43.116    3.5783    1.0000    5554.1     0.1497E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 42.667    3.5663    1.0000    5554.1     0.1783E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 42.227    3.5542    1.0000    5554.1     0.2077E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 41.796    3.5421    1.0000    5554.1     0.2378E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 41.374    3.5299    1.0000    5554.1     0.2680E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 40.960    3.5176    1.0000    5554.1     0.2980E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 40.554    3.5053    1.0000    5554.1     0.3275E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 40.157    3.4930    1.0000    5554.1     0.3562E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 39.767    3.4807    1.0000    5554.1     0.3840E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 39.385    3.4684    1.0000    5554.1     0.4105E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 39.010    3.4561    1.0000    5554.1     0.4357E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 38.642    3.4438    1.0000    5554.1     0.4595E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 38.280    3.4315    1.0000    5554.1     0.4817E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 37.926    3.4193    1.0000    5554.1     0.5023E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 37.578    3.4071    1.0000    5554.1     0.5214E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 37.236    3.3950    1.0000    5554.1     0.5389E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 36.901    3.3830    1.0000    5554.1     0.5549E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 36.571    3.3710    1.0000    5554.1     0.5694E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 36.248    3.3592    1.0000    5554.1     0.5825E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 35.930    3.3474    1.0000    5554.1     0.5944E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 35.617    3.3358    1.0000    5554.1     0.6053E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 35.310    3.3242    1.0000    5554.1     0.6151E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 35.009    3.3128    1.0000    5554.1     0.6244E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 34.712    3.3015    1.0000    5554.1     0.6332E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 34.420    3.2904    1.0000    5554.1     0.6419E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 34.133    3.2794    1.0000    5554.1     0.6510E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 33.851    3.2686    1.0000    5554.1     0.6609E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 33.574    3.2579    1.0000    5554.1     0.6720E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 33.301    3.2474    1.0000    5554.1     0.6847E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 33.032    3.2371    1.0000    5554.1     0.6998E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.768    3.2269    1.0000    5554.1     0.7174E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.508    3.2169    1.0000    5554.1     0.7383E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.252    3.2072    1.0000    5554.1     0.7625E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.000    3.1976    1.0000    5554.1     0.7905E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 31.752    3.1882    1.0000    5554.1     0.8224E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 31.508    3.1790    1.0000    5554.1     0.8581E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 31.267    3.1701    1.0000    5554.1     0.8976E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 31.030    3.1614    1.0000    5554.1     0.9404E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.797    3.1529    1.0000    5554.1     0.9861E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.567    3.1446    1.0000    5554.1     0.1034E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.341    3.1366    1.0000    5554.1     0.1083E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.118    3.1288    1.0000    5554.1     0.1133E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.898    3.1213    1.0000    5554.1     0.1182E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.681    3.1140    1.0000    5554.1     0.1230E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.468    3.1070    1.0000    5554.1     0.1274E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.257    3.1002    1.0000    5554.1     0.1314E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.050    3.0937    1.0000    5554.1     0.1348E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.845    3.0875    1.0000    5554.1     0.1375E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.643    3.0816    1.0000    5554.1     0.1393E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.444    3.0759    1.0000    5554.1     0.1403E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.248    3.0705    1.0000    5554.1     0.1402E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.055    3.0655    1.0000    5554.1     0.1390E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.864    3.0607    1.0000    5554.1     0.1368E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.676    3.0562    1.0000    5554.1     0.1334E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.490    3.0521    1.0000    5554.1     0.1289E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.307    3.0482    1.0000    5554.1     0.1234E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.126    3.0447    1.0000    5554.1     0.1169E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.947    3.0415    1.0000    5554.1     0.1095E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.771    3.0387    1.0000    5554.1     0.1015E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.597    3.0361    1.0000    5554.1     0.9285E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.426    3.0340    1.0000    5554.1     0.8385E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.256    3.0321    1.0000    5554.1     0.7465E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 26.089    3.0307    1.0000    5554.1     0.6544E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 90.000    3.7258    1.0000    5554.1     0.4284E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 85.000    3.7544    1.0000    5554.1     0.5445E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 80.000    3.7804    1.0000    5554.1     0.6518E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 75.000    3.8020    1.0000    5554.1     0.7416E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 70.000    3.8167    1.0000    5554.1     0.8349E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 65.000    3.8211    1.0000    5554.1     0.9925E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 60.000    3.8106    1.0000    5554.1     0.1160E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 55.000    3.7793    1.0000    5554.1     0.9743E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 50.000    3.7199    1.0000    5554.1     0.3880E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 48.000    3.6865    1.0000    5554.1     0.2001E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 46.000    3.6470    1.0000    5554.1     0.1819E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 44.000    3.6008    1.0000    5554.1     0.9926E-06   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 42.000    3.5478    1.0000    5554.1     0.2235E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 40.000    3.4881    1.0000    5554.1     0.3674E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 38.000    3.4218    1.0000    5554.1     0.4980E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 36.000    3.3500    1.0000    5554.1     0.5918E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 34.000    3.2743    1.0000    5554.1     0.6557E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 32.000    3.1976    1.0000    5554.1     0.7905E-05   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 30.000    3.1248    1.0000    5554.1     0.1160E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 29.000    3.0922    1.0000    5554.1     0.1354E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 28.000    3.0641    1.0000    5554.1     0.1384E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
MFT96 R U T -1 27.000    3.0424    1.0000    5554.1     0.1117E-04   55.204899  163.244598   44.619999   82.970001 0  1 COMMENT
