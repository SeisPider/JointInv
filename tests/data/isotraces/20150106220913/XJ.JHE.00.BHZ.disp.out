SURF96 R U T -1 90.000    3.7258    1.0000    
SURF96 R C T -1 90.000    6.1827    1.0000    
SURF96 R U T -1 85.000    3.7544    1.0000    
SURF96 R C T -1 85.000    5.9661    1.0000    
SURF96 R U T -1 80.000    3.7804    1.0000    
SURF96 R C T -1 80.000    5.7681    1.0000    
SURF96 R U T -1 75.000    3.8020    1.0000    
SURF96 R C T -1 75.000    5.5862    1.0000    
SURF96 R U T -1 70.000    3.8167    1.0000    
SURF96 R C T -1 70.000    5.4178    1.0000    
SURF96 R U T -1 65.000    3.8211    1.0000    
SURF96 R C T -1 65.000    5.2606    1.0000    
SURF96 R U T -1 60.000    3.8106    1.0000    
SURF96 R C T -1 60.000    5.1119    1.0000    
SURF96 R U T -1 55.000    3.7793    1.0000    
SURF96 R C T -1 55.000    4.9684    1.0000    
SURF96 R U T -1 50.000    3.7199    1.0000    
SURF96 R C T -1 50.000    4.8261    1.0000    
SURF96 R U T -1 48.000    3.6865    1.0000    
SURF96 R C T -1 48.000    4.7683    1.0000    
SURF96 R U T -1 46.000    3.6470    1.0000    
SURF96 R C T -1 46.000    4.7093    1.0000    
SURF96 R U T -1 44.000    3.6008    1.0000    
SURF96 R C T -1 44.000    4.6488    1.0000    
SURF96 R U T -1 42.000    3.5478    1.0000    
SURF96 R C T -1 42.000    4.5861    1.0000    
SURF96 R U T -1 40.000    3.4881    1.0000    
SURF96 R C T -1 40.000    4.5208    1.0000    
SURF96 R U T -1 38.000    3.4218    1.0000    
SURF96 R C T -1 38.000    4.4521    1.0000    
SURF96 R U T -1 36.000    3.3500    1.0000    
SURF96 R C T -1 36.000    4.3795    1.0000    
SURF96 R U T -1 34.000    3.2743    1.0000    
SURF96 R C T -1 34.000    4.3024    1.0000    
SURF96 R U T -1 32.000    3.1976    1.0000    
SURF96 R C T -1 32.000    4.2205    1.0000    
SURF96 R U T -1 30.000    3.1248    1.0000    
SURF96 R C T -1 30.000    4.1337    1.0000    
SURF96 R U T -1 29.000    3.0922    1.0000    
SURF96 R C T -1 29.000    4.0888    1.0000    
SURF96 R U T -1 28.000    3.0641    1.0000    
SURF96 R C T -1 28.000    4.0429    1.0000    
SURF96 R U T -1 27.000    3.0424    1.0000    
SURF96 R C T -1 27.000    3.9966    1.0000    
